Gopkg.toml

```diff
@@ -37,8 +37,9 @@
   name = "github.com/EvilSuperstars/go-cidrman"
 
 [[constraint]]
-  revision = "31228935eec685587914528585da4eb9b073c76d"
+  revision = "a27b59fe2be9d16e2b66f0d7acf5c852f844cc2b"
   name = "github.com/altoros/gosigma"
+  source = "github.com/wallyworld/gosigma"
```

Comment:
Should this be "wallyworld" ? I would have expected us to use github.com/juju/gosigma so that it doesn't require any single person to fix it.
Also, I didn't audit the actual branch to make sure it has the right patch applied, but I'm happy with the concept.

Reply:
I applied Cunningham's Law to this :smile: